### PR TITLE
LPS-116025 Side panel close button effect when mouse is hovered

### PR DESCRIPTION
**Pull request by brunofariasdeo**
https://issues.liferay.com/browse/LPS-116025

### CODE REVIEW

**Comment by brunofariasdeo** [comment]
ci:test:sf

**Comment by brunofariasdeo** [comment]
ci:test:relevant

**Comment by brunofariasdeo** [comment]
ci:test:forms

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/sidebar/Sidebar.es.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ export default () => {
 
 								<li className="tbar-item">
 									<ClayButton
-										displayType="unstyled"
+										className="component-action sidebar-close"
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/sidebar/Sidebar.es.js:85]: @brunofariasdeo, can you please give me more details about this fix? How did you decide that these classes were the correct ones to be added?

Comment (brunofariasdeo) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/sidebar/Sidebar.es.js:85]: @natocesarrego, I believed this close button should have the same behaviour and appearance that form builder sidebar has, hence I've use the same class.

**Pull request closed without merging**